src/recommendations/implementation.md

```diff
@@ -0,0 +1,41 @@
+---
+group: product-recommendations
+title: How to Implement Product Recommendations
+ee_only: True
+---
+
+To help you implement product recommendations on your storefront, use the following workflow:
+
+1. **Deploy data collection to production**
+
+   Deploying Product Recommendations requires two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). Because production is the only environment where your shoppers' actions can captured and analyzed, it is in your best interest to start data collection on production as early as possible. The more data you collect by the time you start showing recommendations to shoppers, the higher quality you can expect your recommendations to be. As an added benefit, when you start collecting behavioral data on production, you can [fetch recommendations based on this production data while operating in non-production environments]({{ page.baseurl }}/recommendations/test.html). This allows you to test and experiment with different recommendations that are computed based on real shopper data collected in production.
```

Comment:
The sentence beginning with "The more data" is a little hard to parse. Maybe rewrite the sentence, like "You can expect higher quality recommendations when you have more data."